ecl/hql/hqlgram.y

```diff
@@ -11555,7 +11557,7 @@ nonDatasetExpr
     : expression            
                         {
                             node_operator op = $1.getOperator();
-                            if (op == no_sortlist) //|| (op == no_list))
+                            if (op == no_sortlist || op==no_dedup) //|| (op == no_list))
```

Comment:
I was getting an error when it called parser->normalizeExpression().. can't replicate the error now.  I'm undoing this change.